### PR TITLE
Added: Update Status API

### PR DESCRIPTION
**Pull request by shreyvishal**
## Description
This PR will add update status API and get status API

## Features covered
NO-Ticket Availbale Yest

## Changes made

* Added a new controller for update status and get status

## Screenshots/recordings
NA

## Testing
NA

## Self checks

- [ ] Code follows coding style guidelines.
- [ ] All tests pass successfully.
- [ ] No new linting errors or warnings.
- [ ] Documentation is updated, if necessary.
- [ ] Dependencies are updated, if required.
- [ ] There are no merge conflicts.

## Reviewer Suggestions
NA

### CODE REVIEW

File: controllers/Bot.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import ActionsService from '../services/Actions.js'
 import AI from '../services/AI.js'
-import DBService from '../services/DBService.js'
 import logger from '../utils/logger.js'
+import {db} from '../server.js'
```
Comment (mayurvir) [inline controllers/Bot.js:4]: Why this change?

File: services/DBService.js [hunk 2]
```diff
@@ -149,6 +149,38 @@ class DBService {
         
         return sessions;
     }
+    async set_data(key, data) {
```
Comment (mayurvir) [inline services/DBService.js:152]: what are these functions being used for?

**Review by mayurvir — CHANGES_REQUESTED** [review]

File: controllers/Bot.js (hunk 1 above)
Comment (shreyvishal) [reply controllers/Bot.js:4]: Have reverted this

File: services/DBService.js (hunk 2 above)
Comment (shreyvishal) [reply services/DBService.js]: Removed these function for now but thought, it could be used as a getter and setter
